lbl/instruments/default.py

```diff
@@ -465,7 +465,7 @@ def write_rdb_fits(self, filename: str, rdb_data: Dict[str, Any]):
                       dtype=datatypelist, names=name_list)
 
     def write_template(self, template_file: str, props: dict,
-                       sci_hdict: io.LBLHeader, sci_table: dict):
+                       sci_hdict: io.LBLHeader, sci_table):
```

Comment:
add `sci_table: Dict[str, Any]`


may need to add `Dict` and `Any` to typing import